dlc-wallet/src/main/scala/org/bitcoins/dlc/wallet/models/DLCAnnouncementDAO.scala

```diff
@@ -7,11 +7,15 @@ import slick.lifted._
 
 import scala.concurrent.{ExecutionContext, Future}
 
+private case class DLCAnnouncementPrimaryKey(
```

Comment:
I don't think these will be able to be private, i don't think there is a reason to keep them private. I believe this is causing the compiler errors

https://github.com/bitcoin-s/bitcoin-s/pull/3100/checks?check_run_id=2633397282#step:5:133